.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
This is a devcontainer file that allows for development in a containerized environment. It helps a lot to solve the annoying Python-environment hell. If you use vscode, try Control-P + "Reopen in container" to get a 100% clean environment.